### DISCUSSION TITLE
Is it possible to sync to a different table name on server side?

### DISCUSSION THREAD

**Discussion by 02pheland** [Mimetis/Dotmim.Sync#1176] [Q&A] [answered]
Hello, 

I have a master database which is running a WebApi. In its syncSetup I have a table called Accounts and its set to UploadOnly.
I then have clients running Postgres who are uploading their Accounts table. 
That part is all working fine, but the clients are split into groups, so there might be 10 clients in group A and 5 clients in group B.

In my master database I need to have separate tables for each group such as A.Accounts and B,Accounts etc.
Is it possible to catch this server side somehow and redirect them to the different table name without changing the client side schema?

I know I can set a schema for each table in the SyncSetup but I need the all of the Postgres tables to remain named the same due to other applications accessing them.

**Comment by Mimetis** [comment, marked as answer]
No, I'm sorry, each table should have its relevant table on both side.

**Comment by 02pheland** [comment]
Ok, thank you for the answer.
